pkgs/development/python-modules/symspellpy/default.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+
+, pythonOlder
+
+, pytestCheckHook
+
+, setuptools
+
+  # for testing
+, numpy
+, importlib-resources
+
+  # requirements
+, editdistpy
+}:
+
+buildPythonPackage rec {
+  pname = "symspellpy";
+  version = "6.7.7";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  patches = [
+    # patch for fix tests
+    (fetchpatch {
+      name = "fix-pkg-resources-deprecation-warning.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/mammothb/symspellpy/pull/150.patch";
+      hash = "sha256-ppWSFAjz5UIpSczsjbXE4MrLloxyPkrVNT/rkAsyC/k=";
+    })
+  ];
+
+  src = fetchFromGitHub {
+    owner = "mammothb";
+    repo = "symspellpy";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-D8xdMCy4fSff3nuS2sD2QHWk0869AlFDE+lFRvayYDQ=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    editdistpy
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    numpy
+    importlib-resources
+  ];
+
+  pythonImportsCheck = [
+    "symspellpy"
+    "symspellpy.symspellpy"
+  ];
+
+  meta = with lib;
+    {
+      description = "symspellpy is a Python port of SymSpell v6.7.1, which provides much higher speed and lower memory consumption";
```

```suggestion
      description = "Python port of SymSpell v6.7.1, which provides much higher speed and lower memory consumption";
```
Please don't repeat the package name.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description